### PR TITLE
Update the README.md file to describe the repo.

### PR DESCRIPTION
**Pull request by TMazenge**
Updated README.md describing the purpose of the repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,5 @@
...
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repository contains Timothy Mazenge's portfolio and STEP Project.
+It will involve the learning of HTML, CSS and JavaScrtipt.
+In addition, it will include building the back end of  a website using Java Servlets, Datastore, and JSON.
```
Comment (ankitaj224) [inline README.md:5]: nit: Correct Spacing

File: portfolio/src/main/webapp/about.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/about.html:5]: nit: Timothy's

File: portfolio/src/main/webapp/about.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                <div class="about-img">
+                    <img src="images/tim.jpg" style="width:75%;height:75%;">         
+                </div>    
+            </div>
+
+             <div class="my-about-text-item">
+                <div class="about-text">
+                    <h1>About me</h1> 
+                    <p>My name is Timoithy Mazenge a rising junior at Fisk University at Fisk University Nashville, Tennesse. I grew up in Zimbabwe which is in the southern of Africa. Growing up I was always
+                       fascinated with America and the Ammerican Dream. As such i was motivated to work extremely hard as a way to chase after my American Dream.<p>
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/about.html:43]: nit: American

File: portfolio/src/main/webapp/about.html [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+                    <h3>Time Till Graduation</h3>
+                    <h3 id="grad"></h3>
+                </div> 
+            </div>
+
+             <div class="my-about-content-item">
+                <div class="about-img">
+                    <img src="images/africa-3.jpg" style="width:75%;height:75%;">
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/about.html:51]: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

Add alt

File: portfolio/src/main/webapp/about.html [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+             <div class="my-about-content-item">
+                <div class="about-img">
+                    <img src="images/africa-3.jpg" style="width:75%;height:75%;">         
+                </div>    
+            </div>
+
+             <div class="my-about-text-item">
+                <div class="about-text">
+                    <h1>Origin</h1> 
+                    <p>I come from Zimabwe which is in Southern Africa<p>
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/about.html:58]: nit:.

File: portfolio/src/main/webapp/script.js [hunk 6]
```diff
@@ -12,17 +12,66 @@
...
+images[6] = "images/photography-1.jpg";
+images[7] = "images/photography.jpg";
+images[8] = "images/photo-3.jpg";
+images[9] = "images/photo2.jpg";
+
+images[10] = "images/art.jpg";
+images[11] = "images/art4.jpg";
+images[12] = "images/art5.jpg";
+
+
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/script.js:38]: Descriptive comments would be helpful to understand the functionality

Comment (TMazenge) [reply portfolio/src/main/webapp/script.js:38]: Noted, I will add descriptive comments.

File: portfolio/src/main/webapp/mission.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    <div id="my-header">
+      <h1>My Life's Mission</h1> 
+    </div>
+
+     <div class="my-tabs">
+        <nav> 
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="activities.html">Extracurrilucar Activities</a></li>
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/mission.html:20]: nit: Extracurricular

File: portfolio/src/main/webapp/destination.html [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+    <div id="my-header">
+      <h1>Travel Destinations</h1> 
+    </div>
+
+     <div class="my-tabs">
+        <nav> 
+            <ul>
+                <li><a href="index.html">Home</a></li>
+                <li><a href="about.html">About</a></li>
+                <li><a href="activities.html">Extracurrilucar Activities</a></li>
```
Comment (ankitaj224) [inline portfolio/src/main/webapp/destination.html:20]: nit: Spell Check

**Pull request merged**